### DISCUSSION TITLE
Auto publishing to NPM

### DISCUSSION THREAD

**Discussion by taras** [jsr-io/jsr#743] [Ideas]
On the [Changelog podcast](https://changelog.com/podcast/610), @ry mentioned that the Deno team is considering adding support for automatically publishing to NPM. I couldn't find an issue or a discussion, so I created this to show my support. For the [Effection project](https://github.com/thefrontside/effection), we'd like to create a contrib monorepo, with a bunch of reusable operations. We already publish our main project to NPM & Deno Land and we're planning to start publishing to JSR. It would be great if we could just publish to JSR and have it automatically propagate from there.

**Comment by nshiab** [comment]
This would be great!

**Comment by AugustinMauroy** [comment]
it's on roadmap of JSR but for now I can invite you to take a look at https://github.com/AugustinMauroy/typescript-package-template
